test/OpenTelemetry.Tests/BaseExporterTest.cs

```diff
@@ -0,0 +1,62 @@
+// <copyright file="BaseExporterTest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+using Xunit;
+
+namespace OpenTelemetry.Tests
+{
+    public class BaseExporterTest
+    {
+        [Fact]
+        public void Verify_ForceFlush_HandlesException()
+        {
+            // By default, ForceFlush should return true.
+            var testExporter = new DelegatingExporter<object>();
```

Comment:
Is this actually testing the return value of `BaseExporter<T>.ForceFlush()`? This would test what `DelegatingExporter.OnForceFlushFunc` returns.